apisix/consumer.lua

```diff
@@ -32,6 +33,23 @@ local lrucache = core.lrucache.new({
     ttl = 300, count = 512
 })
 
+
+local function decrypt_items(name, conf)
+    local schema = plugin.get(name)
+    local consumer_schema = schema.consumer_schema
+    if not consumer_schema then
+        return
+    end
+
+    for key, props in pairs(consumer_schema.properties) do
+        if props.type == "string" and props.encrypted then
+            local encrypted = apisix_ssl.aes_decrypt_pkey(conf[key], "global_data_encrypt")
+            conf[key] = encrypted
+        end
+    end
```

Comment:
The schema may have nested objects

Reply:
I know that since this PR does not include this case, we can optimize this point in the next PR.